tools/deploy/eda-server/deployment.yaml

```diff
@@ -29,5 +29,12 @@ spec:
           ports:
             - containerPort: 8080
           resources: {}
+          volumeMounts:
+            - name: dockersock
+              mountPath: "/var/run/docker.sock"
+      volumes:
+        - name: dockersock
+          hostPath:
+            path: /var/run/docker.sock
```

Comment:
I think this part has no relation with the PR, am i wrong?

Reply:
good catch